src/auth/frappe.auth.controller.ts

```diff
@@ -408,6 +409,19 @@ export default class FrappeAuthController extends AuthController {
       : RequestResponse.fail(this.handleError("logout", r.error));
   }
 
+  public async setSessionStatusInfo({
```

Comment:
For the sake of consistency, I reckon this method should be in `auth.controller.ts`.

Reply:
if you check auth.controller.ts

`
 public abstract async setSessionStatusInfo({
    sessionStatusInfo
  }: {
    sessionStatusInfo: SessionStatusInfo;
  });

`

just like logout(),verifyOTP etc..